docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -100,67 +100,67 @@ After attempting to deploy {WebNameTitle} on OpenShift, the Pod is unable to sta
 
 *What it means:*
 
-The resource quota for the OpenShift project has been met, and the Pod is unable to obtain the requested resources.
+The resource quota for the OpenShift project has been met, and the pod is unable to obtain the requested resources.
 
 *How to resolve it:*
 
-Perform either of the following:
+Do either of the following:
 
-* Increase the quota for the OpenShift project. For additional information, see link:{OpenShiftDocsURL}/applications/quotas/quotas-setting-per-project.html[Quotas and Limit Ranges] and link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-limit-ranges.html[Setting Limit Ranges] in the OpenShift documentation.
+* Increase the quota for the OpenShift project. For additional information on OpenShift quotas, see link:{OpenShiftDocsURL}/applications/quotas/quotas-setting-per-project.html[Quotas and Limit Ranges] and link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-limit-ranges.html[Setting Limit Ranges].
 * Reduce the requested resources for the {WebName} OpenShift project. It is recommended to have at least 2 CPUs and 4 GB of memory for the project.
 
 Once the request is within the available quota, attempt the deployment once again.
 
-.Pod takes longer than 600 seconds to become available
+.A pod takes longer than 600 seconds to become available
 
-After attempting a deployment, the `-deploy` Pods time out and report the following error.
+After attempting a deployment, the `-deploy` pods time out and report the following error:
 
 [source,options="nowrap",subs="+quotes"]
 ----
 error: update acceptor rejected mta-web-console-executor-1: Pods for rc 'mta/<POD_NAME>' took longer than 600 seconds to become available
 ----
 
-These errors appear after the Pods time out and are placed into an error state.
+These errors appear after the pods time out and are placed into an error state.
 
 *What it means:*
 
-The deployment Pods are unable to successfully launch the Pods.
+The `-deploy` pods are unable to successfully launch the pods.
 
 This error can be caused by a number of sources, with the following being the most common:
 
```

Comment:
fluff. Change to: This error might be caused by insufficient cluster resources or unsuccessful attempts to pull images from the registry.